### PR TITLE
Object Detection UX & Documentation updates

### PR DESCRIPTION
**Pull request by Advitya17**
## Description

This PR adds a callout to accommodate the definition for the IoU score, hides the heatmap toggle from the feature cohort view, and updates the documentation for the metrics.

image

image

image

image

image

## Checklist

- [x] I have added screenshots above for all UI changes.
- [ ] I have added e2e tests for all UI changes.
- [x] Documentation was updated if it was needed.

### CODE REVIEW

File: libs/localization/src/lib/en.json [hunk 1]
```diff
@@ -1774,7 +1774,13 @@
         "micro": "Micro"
       },
       "classSelectionDropdown": "Select class(es)",
-      "iouthresholdDropdown": "IoU Threshold",
+      "iouthresholdDropdown": {
+        "name": "IoU Threshold",
+        "description": "The Intersection over Union score is calculated as the ratio of area of intersection to area of union of the ground truth and predicted bounding boxes. A higher IoU score means the predicted box needs to more closely resemble the ground truth bounding box to be deemed a correct prediction.",
```
Comment (hawestra) [inline libs/localization/src/lib/en.json:1779]: nit: should intersection and union be capitalized ?

File: libs/localization/src/lib/en.json [hunk 2]
```diff
@@ -1774,7 +1774,13 @@
         "micro": "Micro"
       },
       "classSelectionDropdown": "Select class(es)",
-      "iouthresholdDropdown": "IoU Threshold",
+      "iouthresholdDropdown": {
+        "name": "IoU Threshold",
```
Comment (hawestra) [inline libs/localization/src/lib/en.json:1778]: also not sure if threshold should be capitalized

**Review by hawestra — APPROVED** [review]

File: libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ObjectDetectionModelOverview.tsx [hunk 3]
```diff
@@ -96,7 +97,12 @@ export class ObjectDetectionWidgets extends React.PureComponent<IObjectDetection
           valueFormat={(value: number): string => `IoU=${value}%`}
           showValue
         />
-      </Stack.Item>
+        <InfoCallout
```
Comment (imatiach-msft) [inline libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ObjectDetectionModelOverview.tsx:100]: this looks a bit far out, maybe you should put it next to the text/label instead?

**Review by imatiach-msft — APPROVED** [review]

File: libs/localization/src/lib/en.json (hunk 1 above)
Comment (Advitya17) [reply libs/localization/src/lib/en.json:1779]: Kept it as it is as it's mentioned as `area of intersection/union`

File: libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ObjectDetectionModelOverview.tsx (hunk 3 above)
Comment (Advitya17) [reply libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ObjectDetectionModelOverview.tsx:100]: Curious on how I should do that? It's displayed to the right of the widget, and still close to `IoU=x%`.

I also tried displaying it below the IoU threshold:

image

Let me know your preference or if there's a way to still bring it closer to the slider title. Thanks!

**Pull request merged**